pkg/result.go

```diff
@@ -46,3 +48,45 @@ func ChaosEngineVerdict(testsDetails *types.TestDetails, clients environment.Cli
 
 	return nil
 }
+
+// ChaosRunHistoryVerdict checks the chaos result run history according to the ChaosResultVerdict
+func ChaosRunHistoryVerdict(testsDetails *types.TestDetails, clients environment.ClientSets, previousRunHistory v1alpha1.HistoryDetails) error {
```

Comment:
Though the comment is provided, I would still recommend a different name for this function.
Maybe `CheckRunHistoryUpdate` cc: @uditgaurav

Reply:
This can be done -- @avaakash!